fastapi/dependencies/_aliases.py

```diff
@@ -0,0 +1,42 @@
+from inspect import Parameter
+from typing import Any, Dict, Optional, Type, TypeVar, cast
+
+from fastapi.params import Deferred, Depends
+from fastapi.requests import Request
+from typing_extensions import Annotated, get_args, get_origin
+
+T = TypeVar("T")
+
+
+LifespanState = Dict[str, Any]
+
+
+def get_type(param: Parameter) -> Type[Any]:
+    annotation = param.annotation
+    while get_origin(annotation) is Annotated:
+        annotation = get_args(annotation)[0]
+    return annotation  # type: ignore[no-any-return]
```

Comment:
There probably is (or maybe could be) a utility function somewhere else that does this and other manipulation of `Annotated`.